src/rpc/rpccategory.cpp

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) 2009-2010 Satoshi Nakamoto
+// Copyright (c) 2009-2018 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#include <rpc/rpccategory.h>
+
+namespace rpccategory
+{
+
+std::string Label (const RPCCategory category);
+
+std::string Label (const RPCCategory category)
+{
+    #ifdef HANDLE_CASE_RETURN
+    #error "HANDLE_CASE_RETURN already defined"
```

Comment:
I don't think this is necessary

Reply:
you will miss it, in case of unpredictable mistakes ;-)